interfaces/builtin/steam_support.go

```diff
@@ -197,7 +197,8 @@ umount /,
 deny /usr/bin/{chfn,chsh,gpasswd,mount,newgrp,passwd,su,sudo,umount} x,
 /usr/bin/** ixr,
 /usr/sbin/** ixr,
-/usr/lib/pressure-vessel/** ixr,
```

Comment:
Is this removal of `/usr/lib/pressure-vessel/` intended? Sorry, maybe I miss some context here but it seems not super obvious to me, maybe a small line in the PR description why it is no longer needed?

Reply:
It's redundant with `/usr/lib/** ixr`, which is a strictly broader permission.

Reply:
Silly me, indeed